source/settings/config-file.txt

```diff
@@ -126,6 +126,15 @@ Settings
        option but doesn't prevent users from manually enabling the
        option with the :guilabel:`Edit connection string` toggle. 
 
+   * - proxy
+     - Allows you to enforce or define a Proxy to connect to your deployment 
+       with. For more information on connecting to your deployment with a
+       proxy, see :ref:`ssh-connection`.
+
+       You must pass either a URL or a DevtoolsProxyOptions object as JSON
+       for the Proxy setting. You can also set the Proxy setting to an empty
+       string. 
```

Comment:
Can we specify that if `proxy` is set to an empty string, no proxy will be used?